tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.AppSec.cs

```diff
@@ -114,6 +114,11 @@ internal class AppSec
             /// Activate debug logs for the WAF
             /// </summary>
             internal const string WafDebugEnabled = "DD_APPSEC_WAF_DEBUG";
+
+            /// <summary>
+            /// Activate debug logs for the WAF
```

Comment:
Copy-pasta, could we describe what SCA is here too? 😄

Reply:
fixed in 63462c108